### PR TITLE
Control enable-bit-vectors default value.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

File: flags/src/main/java/com/yahoo/vespa/flags/Flags.java [hunk 1]
```diff
@@ -383,6 +383,13 @@ public class Flags {
             "Takes effect at redeployment",
             ZONE_ID, APPLICATION_ID);
 
+    public static final UnboundBooleanFlag ENABLE_BIT_VECTORS = defineFeatureFlag(
+            "enable-bit-vectors", false,
+            List.of("arnej"), "2022-05-03", "2022-12-31",
```
Comment (geirst) [inline flags/src/main/java/com/yahoo/vespa/flags/Flags.java:388]:
```suggestion
            List.of("baldersheim"), "2022-05-03", "2022-12-31",
```

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java [hunk 2]
```diff
@@ -118,6 +118,7 @@ interface FeatureFlags {
         @ModelFeatureFlag(owners = {"bjorncs", "baldersheim"}) default boolean enableJdiscPreshutdownCommand() { return true; }
         @ModelFeatureFlag(owners = {"arnej"}) default boolean avoidRenamingSummaryFeatures() { return false; }
         @ModelFeatureFlag(owners = {"arnej"}) default boolean experimentalSdParsing() { return false; }
+        @ModelFeatureFlag(owners = {"balder"}) default boolean enableBitVectors() { return false; }
```
Comment (geirst) [inline config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java:121]:
```suggestion
        @ModelFeatureFlag(owners = {"baldersheim"}) default boolean enableBitVectors() { return false; }
```

File: config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java [hunk 3]
```diff
@@ -77,6 +77,7 @@ public class TestProperties implements ModelContext.Properties, ModelContext.Fea
     private List<String> environmentVariables = List.of();
     private boolean avoidRenamingSummaryFeatures = true;
     private boolean experimentalSdParsing = true;
+    private boolean enableBitVectors = true;
```
Comment (geirst) [inline config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java:80]: I suggest setting this to _false_ to match the current default.

**Review by geirst — APPROVED** [review]
👍

File: config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java (hunk 3 above)
Comment (baldersheim) [reply config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java]: Done

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**